src/Integration.Vsix/CFamily/PortedFromJava/Analyzer.cs

```diff
@@ -60,7 +60,6 @@ internal partial class Request
         public string File { get; set; } = "";
     }
 
-    [Serializable]
```

Comment:
This attribute relates to the framework binary serializer, not the Newtonsoft serializer. It suggests we use binary serialization, which we don't.

Reply:
Thanks for the fix 👍